docs/source/conf.py

```diff
@@ -80,6 +80,12 @@
 #  "**": [],
 #}
 
+# Ignore links that do not work with github actions link checking
+# https://github.com/neuroinformatics-unit/actions/pull/24#issue-1978966182
+linkcheck_ignore  = [
+    "https://neuroinformatics.zulipchat.com/#narrow/stream/405999-DataShuttle"
+]
```

Comment:
I think you would still want to check that the base URL works, and only skip checking the anchors, rather than ignoring individual full URLs with `linkcheck_ignore`?: 
```suggestion
linkcheck_anchors_ignore_for_url = [
    "https://neuroinformatics.zulipchat.com/"
]
```

Reply:
+1. I think you need to add the base URL in `linkcheck_anchors_ignore_for_url`, as in @lochhh's suggestion. In your earlier commit, you had added the full path to the zulip channel, which wouldn't work I think.

Reply:
Thanks both, I thought I tried this locally and it still errored, but maybe I had not organised it in this way. Will check on the CI, cheers!

Reply:
Hey @lochhh @niksirbi indeed for some reason this does not work, do you have any idea why? seems like it should 🤔

Reply:
from the error output, it seems like sphinx is outdated: sphinx-5.3.0 
`linkcheck_anchors_ignore_for_url` is [new in version 7.1](https://www.sphinx-doc.org/en/master/usage/configuration.html#:~:text=tuple%20().-,New%20in%20version%207.1.,-linkcheck_auth)

Reply:
Awesome thanks @lochhh that did it!! 🙏 🙏